test/integration/install-test.js

```diff
@@ -143,4 +140,36 @@ describe(install, function() {
 
     assertNoStaged(status);
   });
+
+  it('can install an addon with a custom blueprint that does not match package name', async function() {
+    let addonContainingBlueprint = 'test/fixtures/blueprint/addon/default-blueprint-different-than-name';
+    // In the package.json, the package version value will be 'file:../legacy-blueprint'
+    let copiedAddonPackagePath = '../legacy-blueprint';
```

Comment:
This doesn't have to be hardcoded, we can get it from test/fixtures/blueprint/addon/legacy-app/merge/my-app-install-blueprint-different-than-name.

Reply:
changed to:
```

    let {
      location
    } = (await loadSafeBlueprintFile('test/fixtures/blueprint/addon/legacy-app/merge/my-app-install-blueprint-different-than-name/config/ember-cli-update.json')).blueprints[1];
```